### DISCUSSION TITLE
CMake Support

### DISCUSSION THREAD

**Discussion by jacking75** [cabinpkg/cabin#774] [Ideas]
I am using C ++ IDE CLion and must use CMake.
I wonder if you can use poac in CMake. If so, how should I use it?

**Comment by ken-matsui** [comment]
Currently, you may not use CMake and only poac internal build system can be used.
But I work positively to this problem, so I planning to include that feature in the next release.

**Comment by jacking75** [comment]
I will be happy to see CMake support in the next release.
thank you.
